parlai/agents/starspace/starspace.py

```diff
@@ -386,9 +381,8 @@ def valid(obs):
 
         max_x_len = max([len(x) for x in parsed_x])
         for x in parsed_x:
-            x += [[self.NULL_IDX]] * (max_x_len - len(x))
+            x += [self.NULL_IDX] * (max_x_len - len(x))
```

Comment:
are you sure about this change? :)

Reply:
I'm pretty sure this is correct; `x` is a `deque` of `int`s here, so it would not make sense to add a list of a lists to a deque of ints (rather, we should add a list of ints to a deque of ints)